src/main/java/com/exasol/sql/expression/rendering/ValueExpressionRenderer.java

```diff
@@ -4,124 +4,234 @@
 import com.exasol.sql.ColumnsDefinition;
 import com.exasol.sql.UnnamedPlaceholder;
 import com.exasol.sql.expression.*;
+import com.exasol.sql.expression.comparison.Comparison;
+import com.exasol.sql.expression.comparison.ComparisonVisitor;
+import com.exasol.sql.expression.comparison.LikeComparison;
+import com.exasol.sql.expression.comparison.SimpleComparison;
+import com.exasol.sql.expression.function.AbstractFunction;
 import com.exasol.sql.expression.function.Function;
+import com.exasol.sql.expression.function.FunctionVisitor;
 import com.exasol.sql.expression.function.exasol.CastExasolFunction;
 import com.exasol.sql.expression.function.exasol.ExasolFunction;
 import com.exasol.sql.expression.function.exasol.ExasolUdf;
+import com.exasol.sql.expression.literal.*;
```

Comment:
Interesting. This is folded, the four comparisons above are not. Looks like the folding rules are inconsistent.